Btu-building-philadelphia/css/style.css

```diff
@@ -0,0 +1,119 @@
+html {
+    font-family: sans-serif;
+}
+
+main {
+    display: flex;
+    flex-direction: row;
+}
+
+#map {
+    height: 580px;
+    width: 100%;
+}
+
+.left-side {
+    border: 1px solid rgba(0, 0, 255, 0);
+}
+
+.address-section {
+    border: 1px solid rgba(0, 128, 0, 0);
+    width: 400px;
+    
+}
+
+.type-section {
+    border: 1px solid rgba(232, 13, 13, 0);
+    margin-bottom: 1rem;
+}
+
+#building-filters {
+    column-count: 3;
+    margin-right: 1rem;   
+}
+
+label {
+    display: block;
+}
+
+[name="school-filter-name"] {
+    display: block;
+    column-span: all;
+    box-sizing: border-box;
+    padding: 0.5rem;
+    margin-top: 0.5rem;
+    width: 100%;
+}
+
+.building-list-section {
+    border: 1px solid rgba(255, 166, 0, 0);
+    margin-top: 0.5rem;
+    height: 473px;
+    overflow-y: auto;
+}
+
+.building-list {
+    padding: 0;
+    list-style: none;
+
+}
+
+.building-name {
+    font-weight: bold;
+    margin-bottom: 0.3rem;
+}
+
+.building-address {
+    font-size: 0.8rem;
+    margin-bottom: 0.3rem;
+
+}
+
+.building-EUI,
+.building-type {
+    font-size: 0.8rem;
+    display: inline;
+    border: 1px solid gray;
+    border-radius: 0.2rem;
+    padding: 0.2rem;
+    background-color: #eee;
+}
+
+.building-EUI-high {
+    font-size: 0.8rem;
+    color: azure;
+    display: inline;
+    border: 1px solid gray;
+    border-radius: 0.2rem;
+    padding: 0.2rem;
+    background-color: #ff0000;
```

Comment:
**accessibility suggestion:** The Axe developer tools raise the issue that the color contrast ratio on your `.building-EUI-high` labels is just barely too low (basically the foreground doesn't stand out from the background well enough). For best accessibility, the [WCAG](https://www.w3.org/TR/WCAG21/) recommends a contrast ratio of at least 4.5:1. This is a handy tool for checking color contrast: https://webaim.org/resources/contrastchecker/. If you put in your red (`#ff0000`) and `azure` (`#f0ffff`) you'll see the contrast ratio is 3.89. The easiest thing to do in this situation is just darken the shade of red a bit.
```suggestion
    background-color: #eb0000;
```